_includes/install/allowoverrides24.md

```diff
@@ -7,9 +7,10 @@ Failure to enable these settings typically results in no styles displaying on yo
 
 1.  Enable the Apache rewrite module:
 
-		```bash
-		a2enmod rewrite
-		```
+	
```

Comment:
This block needs to be indented one space. See the next step for an example of correct alignment.

Reply:
Sure jeff. next time i will take care. Original i have try to solve. [https://ibb.co/hnwjSK](url) . 

but this issue still there in that page. can you resolve please resolve it.